music21/spanner.py

```diff
@@ -1166,7 +1341,9 @@ class MultiMeasureRest(Spanner):
     }
 
     def __init__(self, *spannedElements, **keywords):
+        from music21 import note
         super().__init__(*spannedElements, **keywords)
+        self.fillElementTypes = [note.Rest]
```

Comment:
Yeah, if only it actually worked... In the latest push, Ottavas are the only Spanner that actually can be filled, because it's the only Spanner that we need to fill inside of music21 (and in my music21 parsers) to get Ottava transposition working. My goal here is to get all this in before v9 is released, so I'm trying to lop off unused stuff that might take too long to get right. Please let me know if there are particular other-than-Ottava Spanners you want fillable in v9.